hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/ContainerController.java

```diff
@@ -159,6 +159,29 @@ public void closeContainer(final long containerId) throws IOException {
     getHandler(container).closeContainer(container);
   }
 
+  /**
+   * Returns the Container given a container id.
+   *
+   * @param containerId ID of the container
+   * @return Container
+   */
+  public void addFinalizedBlock(final long containerId,
+                 final long localId) {
```

Comment:
Wrapped line indent is 4.  BTW, the max line length is 120 now, not 80 any more.